### DISCUSSION TITLE
Had been running, stopped... Probably obvious user error

### DISCUSSION THREAD

**Discussion by kubycsolutions** [automatic-ripping-machine/automatic-ripping-machine#1214] [Q&A]
As of about a week and a half ago:
```arm@medea:~$ ./start_arm_container.sh
docker: Error response from daemon: create <path_to_arm_user_home_folder>: "<path_to_arm_user_home_folder>" includes invalid characters for a local volume name, only "[a-zA-Z0-9][a-zA-Z0-9_.-]" are allowed. If you intended to pass a host directory, use absolute path.
See 'docker run --help'.
arm@medea:~$ 
```

I don't _think_ I changed the start configuration...
```docker run -d \
    -p "8080:8080" \
    -e ARM_UID="1001" \
    -e ARM_GID="1001" \
    -v "<path_to_arm_user_home_folder>:/home/arm" \
    -v "<path_to_music_folder>:/home/arm/music" \
    -v "<path_to_logs_folder>:/home/arm/logs" \
    -v "<path_to_media_folder>:/home/arm/media" \
    -v "<path_to_config_folder>:/home/arm/config" \
    --device="/dev/sr0:/dev/sr0" \
    --device="/dev/sr1:/dev/sr1" \
    --privileged \
    --restart "always" \
    --name "arm-rippers" \
    --cpuset-cpus='2,3' \
    automaticrippingmachine/automatic-ripping-machine:latest
```

OK, what bit of idiocy am I committing?

**Comment by kubycsolutions** [comment]
No clues? I'm sure I'm being an idiot, but I honestly do not see what's wrong here. It's as if the -v substitution isn't occurring when I execute `start_arm_container.sh` to run the docker command.

'''
arm@medea:~$ ./start_arm_container.sh
docker: Error response from daemon: create <path_to_config_folder>: "<path_to_config_folder>" includes invalid characters for a local volume name, only "[a-zA-Z0-9][a-zA-Z0-9_.-]" are allowed. If you intended to pass a host directory, use absolute path.
See 'docker run --help'.
'''

**Comment by darkspirit510** [comment]
Sorry to dig out this thread (looking for a similar problem). Your issue is the path, you did not change it or it got reverted. `<>` indicate that you have to add a actual path in there, which you didn't.
